hw/xbox/nv2a/nv2a_pgraph.c

```diff
@@ -3083,6 +3134,23 @@ static void pgraph_bind_shaders(PGRAPHState *pg)
 
     glUseProgram(pg->shader_binding->gl_program);
 
+    /* Clipping regions */
+    for (i = 0; i < state.psh.window_clip_count; i++) {
```

Comment:
Seems to be adding an artifact (in Halo:CE at least, where we are clipping a 1-pixel line around the top and right of the screen. Likely an off-by-1 problem somewhere here.